tests/test_fixity.py

```diff
@@ -74,31 +76,23 @@ def test_scan(_get, mock_check_fixity):
     ]
 
 
-@mock.patch("fixity.utils.utcnow")
 @mock.patch("requests.get")
-def test_scan_if_timestamps_argument_is_passed(_get, utcnow, mock_check_fixity, capsys):
+@mock.patch.object(fixity, "Session", lambda: SESSION)
+def test_scan_if_timestamps_argument_is_passed(
+    _get, monkeypatch, mock_check_fixity, caplog
```

Comment:
As I mentioned in Slack I think this tests requires you to use the `capsys` fixture instead of `caplog` so you can assert that the timestamp is indeed printed in your messages. I think you still need to set a fixed timestamp like we do in the `master` branch and assert your message against that the same way.